### ISSUE TITLE
Fix `aria-current` issue

### ISSUE THREAD

**Issue by tempertemper** [tempertemper/tempertemper.net#719]
in the navigation templates, the `aria-current` attribute is only added when `item.url` and `page.url` match. There's a mismatch when a URL doesn't have a trailing slash, and on these pages the `aria-current` attribute isn't added.
